code/modules/vore/appearance/sprite_accessories_taur_vr.dm

```diff
@@ -45,8 +45,13 @@
 	buckle_lying = FALSE
 
 /mob/living/carbon/human/New()
-	..()
-	riding_datum = new /datum/riding/taur(src)
+	.=..()
+	spawn(2 SECONDS)
```

Comment:
Looks like the `if(isTaurTail...` block ought to be indented to be under the `spawn` statement in order to delay it.
That said, I do not love `spawn` in `New()`.